### DISCUSSION TITLE
When podman 3.1.1 is planned to release in alpine ?

### DISCUSSION THREAD

**Discussion by SahilChaudhary25** [containers/podman#10006] [Q&A]
I tried podman 3.1 rootless and it did not worked for me, looks like issues are being resolved for podman update. When i can expect new version release ?

**Comment by mheon** [comment]
Later this week - I suspect we'll cut the release on Thursday

**Comment by GeorgiaM-honestly** [comment]
> I tried podman 3.1 rootless and it did not worked for me, looks like issues are being resolved for podman update. When i can expect new version release ?

By any chance, are you seeing anything like this? I'm also on alpine. Things worked fine for me until i did a kernel update. Could you kindly take a peek?

https://github.com/containers/podman/discussions/10036 - "container state improper" in Rootless Containers/Pod After Kernel Upgrade and Reboot

@SahilChaudhary25 : Fixed the link, i'm not sure what happened.

**Reply by SahilChaudhary25** [reply]
@GeorgiaM-honestly This page looks broken
